packages/cli/fragments/fragment-clojurescript/.vscode/extensions.json

```
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["tauri-apps.tauri-vscode", "rust-lang.rust-analyzer"]
```

Seems like clojurescript has a vscode extension, so we should recommend that
```suggestion
  "recommendations": ["tauri-apps.tauri-vscode", "rust-lang.rust-analyzer", "betterthantomorrow.calva"]
```